backend/src/modules/sendEmails/application/SenEmailCommandHandler.ts

```diff
@@ -0,0 +1,18 @@
+import { CommandResult } from '../../../shared/Module/core/application/command/CommandResult';
+import { CommandHandler } from '../../../shared/Module/core/application/command/CommandHandler';
+import { SendEmail } from './SendEmailCommand';
+import { EmailSender } from './EmailSender';
+
+export class SendEmailCommandHandler implements CommandHandler<SendEmail> {
+  constructor(private readonly emailSender: EmailSender) {}
```

Comment:
I tu w konstruktorze chcemy odebrać to:
![image](https://user-images.githubusercontent.com/41823689/168164615-27d91b54-a478-422b-8728-0765fb570191.png)

Czyli to co przekażemy z tego miejsca (czyli te publishery oraz generatory):
![image](https://user-images.githubusercontent.com/41823689/168164682-6b86be78-f34e-42f4-90fb-7bba47eca93f.png)

Reply:
👍